package.json

```diff
@@ -19,6 +20,55 @@
     "src",
     "bin"
   ],
+  "exports": {
```

Comment:
I don't think we want this `exports`

Reply:
@fisker I think [current documented example](https://prettier.io/docs/en/browser.html#es-modules) will be broken without `exports`.

I'd remove `exports` if you want even though it's breaking change.